### DISCUSSION TITLE
Dividing ApplicationCommand handlers between channels and bot instances

### DISCUSSION THREAD

**Discussion by 6r1d** [Pycord-Development/pycord#276] [Q&A] [answered]
Hello. I would like to work on the bot debugging in "-dev" channels on the server, while there's a "production" bot.
I want both of them to process slash commands and menu commands.

As far as I understand, Discord will send notification about the interaction to both bots.
From that point on, I can get channel title or description to distinguish if this process should handle the command.
Should I somehow notify Discord the production server isn't handling the command, so another server receives it?

**Comment by middledot** [comment, marked as answer]
You want both bots to process the same slash and context menu commands?

Edit: A registered command will only send interactions to the bot that registered it.

**Reply by 6r1d** [reply]
Yes, I want to do that. One channel description can contain some keyword to mark which instance should do the processing, other should simply ignore some channels.

**Reply by middledot** [reply]
A registered command will only send interactions to the bot that registered it.
